include/shellstrings.h

```diff
@@ -89,31 +122,80 @@ void shell_input(char* input_file, char* status);
  * Returns:
  *  - None.
  */
-void shell_help();
+void shell_help(bool *color);
 
 /*
- * shell_usage - command line usage instructions
+ * shell_start_interactive - prints out interactive starting message
  *
  * Parameters:
+ *  - file, dictionary, mode names
+ *
+ * Returns:
  *  - None.
+ */
+void shell_start_interactive(char *filename, char *dict, char *md, bool *color);
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
changed!